tests/src/gretel_client/unit/test_client.py

```diff
@@ -147,6 +147,30 @@ def records_rev():
     ]
 
 
+@patch("gretel_client.client.Client")
+@patch("gretel_client.client.getpass")
+@patch("gretel_client.client.os.getenv")
+def test_get_cloud_client_prompt(getenv, getpass, Client):
+    # # when no env is set and prompt is true, ask for gretel key
+    getenv.side_effect =None
+    get_cloud_client("api", "prompt")
+    getpass.call_count == 0
```

Comment:
Should this be an assertion? (Same with the ones below it)

Reply:
fixed.